tsl/test/expected/reorder.out

```diff
@@ -289,7 +289,6 @@ COMMIT;
 -- reorder a chunk directly with an chunk index
 SELECT reorder_chunk('_timescaledb_internal._hyper_1_2_chunk', '_timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx', TRUE);
 INFO:  reordering "_timescaledb_internal._hyper_1_2_chunk" using sequential scan and sort
-INFO:  "_hyper_1_2_chunk": found 0 removable, 5 nonremovable row versions in 1 pages
```

Comment:
Here it is reasonable to show that reordering actually executed, so it makes sense to have INFO level events.